samples/msal-node-samples/custom-INetworkModule-and-network-tracing/app.ts

```diff
@@ -4,7 +4,7 @@ import * as msal from "@azure/msal-node";
  * After "npx tsc" is executed via the "npm run start" script, app.ts and HttpClient.ts are compiled to .js and stored in the /dist folder
  * The app is run via "node dist/app.js", hence the .js import of the HttpClient
  */
-// import HttpClient from "./HttpClient.js";
+// import { HttpClient } from "./HttpClient.js";
```

Comment:
If we would like to avoid `.js` imports in `.ts` file without much hassle, I think we could look into using [ts-node](https://www.npmjs.com/package/ts-node) -it's a bit slower than tsc but since this is for development/debugging only, performance is not much of a concern.